01-node-tutorial/answers/02-globals.js

```diff
@@ -0,0 +1,13 @@
+// __dirname  --path to current folder(directory)
+// __filename  --file name
+// moudle --info about  current moudle (file)
+// requier -- function to use moudle (commonJs)
+// process -- info about enviroment where the program is being executed
```

Comment:
use of the environmental variable is missing. Export a variable in the terminal and then print in the terminal to verify if the environment variable was stored.